src/ducks/connections/index.js

```diff
@@ -100,6 +100,12 @@ const reducer = (state = {}, action) => {
               ...((newState[konnectorSlug] &&
                 newState[konnectorSlug].triggers) ||
                 {}),
+              data: [
```

Comment:
Nit: I think using `data` here is only useful for the KonnectorModal, but the redux store should be generic. Maybe a different name or even a slice for the triggers would have been better.